packages/@okta/vuepress-site/docs/api/resources/oidc/index.md

```diff
@@ -1332,33 +1202,72 @@ For more information about configuring an app for OpenID Connect, including grou
 * The full set of claims for the requested scopes is available via the [/oauth2/v1/userinfo](#userinfo) endpoint. Call this endpoint using the access token.
 
 ### Refresh Token
-
 Refresh tokens are opaque. More information about using them can be found in the [Authentication Guide](/authentication-guide/tokens/refreshing-tokens).
 
-## Token Authentication Methods
+## Client Authentication Methods
+When registering an OAuth 2 client application, you can specify an authentication method by including the [token_endpoint_auth_method](https://developer.okta.com/docs/api/resources/apps/#add-oauth-2-0-client-application) parameter. Okta supports five client authentication methods:
+
+> Note: If you don't specify a method when registering your client, the default method is `client_secret_basic`.
+
+* `client_secret_basic`, `client_secret_post`, `client_secret_jwt`: Use one of these three methods to authenticate against the `/token` or `/revoke` endpoints when the client has access to the `client_secret`. Typically services support client authentication via HTTP Basic Auth (`client_secret_basic`) with the client's `client_id` and `client_secret`. However, some services support authentication by accepting the `client_id` and `client_secret` as POST body parameters (`client_secret_post`). Check the service's documentation to find out what the service expects, since the OAuth 2.0 specification leaves this decision to the service.
+
+* `private_key_jwt`: Use this method when you want to use public/private key pairs for more security, as key pairs are short-lived.
```

Comment:
The key pairs are not short lived, it's the JWT which you produce & sign with them that are short lived. The main benefit of using `private_key_jwt` is that it can be a _true_ secret. You can generate your private key in your own servers, and never have it leave there for any reason since you only need to provide the public key to Okta- this is better than `client_secret_jwt` since Okta must know what the client_secret string is beforehand, so there are more places which it could in theory be compromised.